openaev-api/src/main/java/io/openaev/service/detection_remediation/DetectionRemediationAIService.java

```diff
@@ -57,7 +83,7 @@ public DetectionRemediationCrowdstrikeResponse callRemediationDetectionAIWebserv
       String responseBody =
           httpClient.execute(httpPost, response -> EntityUtils.toString(response.getEntity()));
 
-      return mapper.readValue(responseBody, DetectionRemediationCrowdstrikeResponse.class);
+      return (T) mapper.readValue(responseBody, classResponse);
```

Comment:
I don"t think the use of generics in this function is requiered. In both places where callRemediationDetectionAIWebservice is called, the return is saved in a DetectionRemediationAIResponse. So I assume, this function could directly return a DetectionRemediationAIResponse instead of T, and we would not need the cast here. What do you think ?